src/katex.less

```diff
@@ -409,15 +409,10 @@
     .mtable {
         .vertical-separator {
             display: inline-block;
-            margin: 0 -0.025em;
-            border-right: 0.05em solid;
+            // margin and border-right are set in Javascript
```

Comment:
Thanks for making a note here.  Could you include the name of the file?

Reply:
Done.

Reply:
Actually, it's fine.  Keeping filenames in comments up to date is a pain and it's easy enough for people to search for `vertical-separator`.